src-electron/util/args.js

```diff
@@ -271,6 +276,11 @@ For more information, see ${commonUrl.projectUrl}`
 
   env.setSaveFileFormat(ret.saveFileFormat)
 
+  // Set emoji preference via environment variable
+  if (ret.noEmoji) {
+    process.env.NO_EMOJI = '1'
```

Comment:
Best to use the noEmoji option coming from command args i.e. the uc_generate target from pack.json instead of doing a 2 step process of setting this as a environment.